features/factories/claimant_factory.rb

```diff
@@ -47,7 +47,7 @@
     trait :fake_ccd_error_422_claimant do
       title { :"simple_form.options.claimant.title.ms" }
       first_name { 'Force' }
-      last_name { "ErrorUnprocessableEntity-#{SecureRandom.uuid.gsub(/[\d-]/) { |x| (x.ord + 58).chr }}" }
+      last_name { "ErrorUnprocessableEntity #{SecureRandom.uuid.gsub(/[\d-]/) { |x| (x.ord + 58).chr }}" }
```

Comment:
This wont work - this 'special' last name is to be recognised by the fake ccd server as a way to deliberately force an error.  I have changed the way forcing errors work because the old system used characters that fail validation in et1 now.  So if you put the 2 'error handling' errors aside and expect them to fail, we can work on that as a separate issue - and I will make sure the fake ccd server is updated accordingly.

Reply:
I will put this back to as before and reverse this change